imports/plugins/core/catalog/server/no-meteor/queries/findVariantInCatalogProduct.js

```diff
@@ -1,4 +1,7 @@
 /**
+ * @summary Given a catalogProduct from the database, traverses it to find
```

Comment:
Just a note to update this PR with `context.queries.findVariantInCatalogProduct.js` once this is merged: https://github.com/reactioncommerce/reaction/pull/5142/files#diff-d442c3edf8473bfa06bd8fa3bf9d96ddR2